skimage/io/_plugins/tifffile_plugin.py

```diff
@@ -24,7 +24,16 @@ def imread(fname, dtype=None, **kwargs):
     .. [1] http://www.lfd.uci.edu/~gohlke/code/tifffile.py
 
     """
+
     if 'img_num' in kwargs:
         kwargs['key'] = kwargs.pop('img_num')
-    with TiffFile(fname) as tif:
+
+    # separate TiffFile kwargs from other kwargs
+    tiff_keys = ['multifile', 'multifile_close', 'pages', 'fastij', 'is_ome']
+    kwargs_tiff = parse_kwargs(kwargs, *tiff_keys)
```

Comment:
```suggestion
    # parse_kwargs will extract keyword arguments intended for the TiffFile class
    # and remove them from the tiff_keys dictionary in-place
    kwargs_tiff = parse_kwargs(kwargs, *tiff_keys)
```

Reply:
> and remove them from the tiff_keys dictionary in-place

and remove them from the **kwargs** dictionary in-place

Reply:
Oops, thanks @cgohlke! Fixed!